tests/Dialect/TfheRustBool/Emitters/emit_tfhe_rust_bool.mlir

```diff
@@ -3,15 +3,30 @@
 !bsks = !tfhe_rust_bool.server_key
 !eb = !tfhe_rust_bool.eb
 
-// CHECK-LABEL: pub fn test_and(
-// CHECK-NEXT:   [[bsks:v[0-9]+]]: &ServerKey,
-// CHECK-NEXT:   [[input1:v[0-9]+]]: &Ciphertext,
-// CHECK-NEXT:   [[input2:v[0-9]+]]: &Ciphertext,
-// CHECK-NEXT: ) -> Ciphertext {
-// CHECK-NEXT:   let [[v0:.*]] = [[bsks]].and([[input1]], [[input2]]);
-// CHECK-NEXT:   [[v0]]
-// CHECK-NEXT: }
-func.func @test_and(%bsks : !bsks, %input1 : !eb, %input2 : !eb) -> !eb {
-  %out = tfhe_rust_bool.and %bsks, %input1, %input2 : (!bsks, !eb, !eb) -> !eb
-  return %out : !eb
+module{
```

Comment:
FYI, you can actually leave out the `module` wrapper and it will be implicitly added by MLIR at parse time. No changes needed though.